charts/greptimedb-cluster/templates/grafana-dashboards-sidecar.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.dashboards.enabled }}
```

Comment:
I notice it's similar to the current `grafana-dashboards-configmap.yaml`. Can we merge it into one yaml? Maybe you can add some new options to `grafana-dashboards-configmap.yaml`?